src/olympia/reviewers/models.py

```diff
@@ -1146,81 +1146,6 @@ def get_content_review_queue(cls, admin_reviewer=False):
         return qs
 
 
-class RereviewQueueThemeManager(ManagerBase):
-
-    def __init__(self, include_deleted=False):
-        # DO NOT change the default value of include_deleted unless you've read
-        # through the comment just above the Addon managers
-        # declaration/instantiation and understand the consequences.
-        ManagerBase.__init__(self)
-        self.include_deleted = include_deleted
-
-    def get_queryset(self):
-        qs = super(RereviewQueueThemeManager, self).get_queryset()
-        if self.include_deleted:
-            return qs
-        else:
-            return qs.exclude(theme__addon__status=amo.STATUS_DELETED)
-
-
-@python_2_unicode_compatible
-class RereviewQueueTheme(ModelBase):
```

Comment:
Need an issue to remove those tables from the db entirely later

Reply:
https://github.com/mozilla/addons-server/issues/10760